.pre-commit-config.yaml

```diff
@@ -1,6 +1,8 @@
 exclude: |
   (?x)
   # NOT INSTALLABLE ADDONS
+  ^product_state/|
```

Comment:
this seems wrong

Reply:
This one belong to TMP commit which would be removed when pepend PR merged

Reply:
sorry, still makes no sense to me because this module is already in v14 https://github.com/OCA/product-attribute/tree/14.0/product_state